_test/general.test.php

```diff
@@ -5,9 +5,29 @@
  * @group plugin_findologicxmlexport
  * @group plugins
  */
+
+require_once (__DIR__ . '/../DokuwikiXMLExport.php');
+require_once (__DIR__ . '/../admin.php');
 class general_plugin_findologicxmlexport_test extends DokuWikiTest {
 
     /**
+     * A page needs to have content. This is the placeholder for this value.
+     */
+    const PAGE_CONTENT_PLACEHOLDER = 'This page is for test purposes.';
+
+    /**
+     * Remove all DokuWiki pages before each test
+     */
+    public function setUp(){
+        $indexer = new Doku_Indexer();
+        $indexer->clear();
```

Comment:
Does this just clear the indexer or really remove all pages?

Reply:
You are right. The indexer does not help in this case and DokuWiki does not provide a function to delete a page either. In the function `\saveWikiText` there is a part that checks if the page is empty and removes it if this is true. 
New solution would be:

```
{
        $indexer = new Doku_Indexer();
        $pages = $indexer->getPages();
        foreach ($pages as $page) {
            // Saving a page with empty text will result in removing it.
            saveWikiText($page, '', '');
       }
}
```

Reply:
Looks good to me!